src/Neo/SmartContract/Manifest/ContractManifest.cs

```diff
@@ -79,7 +79,8 @@ void IInteroperable.FromStackItem(StackItem stackItem)
             Permissions = ((Array)@struct[5]).Select(p => p.ToInteroperable<ContractPermission>()).ToArray();
             Trusts = @struct[6] switch
             {
-                Null => WildcardContainer<ContractPermissionDescriptor>.CreateWildcard(),
+                Null _ => WildcardContainer<ContractPermissionDescriptor>.CreateWildcard(),
+                Array array when array.Any(p => ((ByteString)p).Size == 0) => WildcardContainer<ContractPermissionDescriptor>.CreateWildcard(),
```

Comment:
This seems to be a wrong fix to me. Just store a proper `Null` and read it, similar to permissions.